Source/Concurrency/RefinementInstrumentation.cs

```diff
@@ -224,7 +224,7 @@ public override List<Cmd> CreateUpdatesToRefinementVars(bool isMarkedCall)
       {
         // assert !pc;
         // pc, ok := true, true;
-        cmds.Add(CmdHelper.AssertCmd(tok, Expr.Not(Expr.Ident(pc)), "Visible state changed before marked call"));
+        cmds.Add(CmdHelper.AssertCmd(tok, Expr.Not(Expr.Ident(pc)), "State modified before marked call"));
```

Comment:
Why not "visible state"?

Reply:
It sounded mysterious to me. You have to understand which part of the state is relevant for refinement checking, and if you don't, I think that "visible" won't help much.

Reply:
Yes, it is mysterious along with myriad of other CIVL concepts. My expectation is that the user of CIVL has read some documentation or is willing to read documentation to understand the meaning of error messages.  

Perhaps, "Either globals or outputs modified before marked call".

Reply:
As discussed, we will call it "layer-n state", where n is the next layer above the disappearing layer of the yielding procedure.